src/lib/scriptEngine.js

```diff
@@ -0,0 +1,92 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * ModusBox
+ - Deon Botha <deon.botha@modusbox.com>
+ - Lazola Lucas <lazola.lucas@modusbox.com>
+ - Claudio Viola <claudio.viola@modusbox.com>
+ --------------
+ ******/
+const MLNumber = require('@mojaloop/ml-number')
+const Transaction = require('../domain/transactions/index')
+const BigNumber = require('bignumber.js')
+const Logger = require('@mojaloop/central-services-logger')
+const ErrorHandler = require('@mojaloop/central-services-error-handling')
+
+const SCRIPT_TIMEOUT = 100
```

Comment:
Please move this to an environmental config (i.e. default.json) with an appropriate name indicating if its seconds or milliseconds.

Reply:
As discussed with @elnyry-sam-k: flag this issue for a future story